tests_peerscout_api/unit_tests/main_test.py

```diff
@@ -189,6 +194,28 @@ def _get_ok_json(response):
     return response.json
 
 
+class TestPickPersonIdFromBqResult:
+    def test_should_return_empty_result_if_passed_list_empty(self):
+        assert not pick_person_id_from_bq_result(
+            bq_person_detail_sql_result=BQ_RESPONSE_LIST,
+            person_ids_to_pick=[]
+        )
+
+    def test_should_return_result_if_passed_one_item_in_list(self):
+        actual_response = pick_person_id_from_bq_result(
+            bq_person_detail_sql_result=BQ_RESPONSE_LIST,
+            person_ids_to_pick=[BQ_RESPONSE_DICT_1['person_id']]
+        )
+        assert actual_response == [BQ_RESPONSE_DICT_1]
+
+    def test_should_return_result_order_if_passed_one_item_in_list(self):
```

Comment:
maybe something like: `test_should_preserve_order_of_passed_in_person_ids`

Reply:
And now you remind me I did not update the latest test name :(